### PR TITLE
Drop deprecated dependency gulp-util

### PR DESCRIPTION
**Pull request by demurgos**
Closes ben-eb/gulp-cssnano#92
Supersedes ben-eb/gulp-cssnano#93

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -79,7 +79,7 @@ test('should throw an error in stream mode', function (t) {
 
     var stream = nano();
 
-    var file = fixture(new Stream());
+    var file = fixture(new PassThroughStream());
```
Comment (phated) [inline test.js]: `new ReadableStream()`

File: test.js [hunk 2]
```diff
@@ -2,8 +2,8 @@
 
 var test = require('tape');
 var nano = require('./');
-var gutil = require('gulp-util');
-var Stream = require('stream');
+var Vinyl = require('vinyl');
+var PassThroughStream = require('stream').PassThrough;
```
Comment (phated) [inline test.js]: `var ReadableStream = require('stream').Readable`

Comment (demurgos) [reply test.js]: Thanks, I'll replace it.

**Comment by vincentmorneau** [comment]
Any news on merging this?

**Comment by phated** [comment]
@vincentmorneau something that worked recently on gulp-modernizr was a community member stepping up and offering to take over the project.  maybe that needs to happen here

**Comment by vincentmorneau** [comment]
@phated What if you or the gulp team would recommend your favorite css minifier? I just saw that gulp-cssnano was not updated in over a year.  

Alternative would be https://github.com/scniro/gulp-clean-css 

Thoughts?

**Comment by phated** [comment]
I don't have a recommendation but didn't postcss have a minifier plugin?  They had a really cool way to do their gulp-plugin that supported many postcss plugins when I looked at it ~2 years ago.

**Comment by demurgos** [comment]
@ben-eb 
Your plugin is still fairly popular. Could you take a look at this PR: merge it and publish it?
In the case of your package the changes are fairly minimal: it just replaces `gulp-util`'s features by their standalone equivalent. This is a patch update from a semver point of view.

**Comment by ben-eb** [comment]
Thanks, I've also rolled in `Buffer.from` support. 👍

**Pull request merged**